hilti/toolchain/src/config.cc.in

```diff
@@ -174,10 +174,10 @@ void Configuration::init(bool use_build_directory) {
 
     hlto_ld_flags_debug = flatten({"-shared", "-Wl,-undefined", "-Wl,dynamic_lookup",
                                    prefix("${HILTI_CONFIG_RUNTIME_LD_FLAGS_DEBUG}", "", installation_tag),
-                                   "-Wno-unused-command-line-argument"});
+                                   "-Wno-unused-command-line-argument", "-Wno-invalid-offsetof"});
     hlto_ld_flags_release = flatten({"-shared", "-Wl,-undefined", "-Wl,dynamic_lookup",
                                      prefix("${HILTI_CONFIG_RUNTIME_LD_FLAGS_RELEASE}", "", installation_tag),
-                                     "-Wno-unused-command-line-argument"});
+                                     "-Wno-unused-command-line-argument", "-Wno-invalid-offsetof"});
```

Comment:
Think `ld_flags` isn't the right place, but the cxx_flags further up?

Splitting the steps in the failing test a bit, it won't succeed with the cuirrent change, but it passes with the flags added to `runtime_cxx_flags_*` instead. So seems to fit better there.
```
-// @TEST-EXEC: $(spicy-config --cxx) -o my_http my_http___linker__.cc my_http_MyHTTP.cc my_http-callback.cc %INPUT $(spicy-config --cxxflags --ldflags)
+// @TEST-EXEC: $(spicy-config --cxx --cxxflags) -c my_http___linker__.cc
+// @TEST-EXEC: $(spicy-config --cxx --cxxflags) -c my_http_MyHTTP.cc
+// @TEST-EXEC: $(spicy-config --cxx --cxxflags) -c my_http-callback.cc
+// @TEST-EXEC: $(spicy-config --cxx --cxxflags) -c %INPUT
+// @TEST-EXEC: $(spicy-config --cxx) -o my_http my_http___linker__.o my_http_MyHTTP.o my_http-callback.o my_http-host-callback.o $(spicy-config --ldflags)
```